tests/bluetooth/controller/ctrl_hci/testcase.yaml

```diff
@@ -0,0 +1,6 @@
+common:
+    tags: test_framework bluetooth bt_ctrl_hci bt_ull_llcp
+tests:
+  bluetooth.controller.ctrl_hci.test:
+     skip: true
```

Comment:
Why is this being skipped?

Reply:
It's being skipped because It doesn't build anymore, the preparation for up-streaming changed how the interfaces were exposed and we didn't have time to fix this prior to up-streaming.